addons/web/static/src/search/cog_menu/cog_menu.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.CogMenu" owl="1">
         <div t-if="hasItems" class="o_cp_action_menus d-flex align-items-center pe-2 gap-1">
-            <Dropdown class="'lh-1'" togglerClass="'d-print-none btn p-0 ms-1 lh-sm'" menuClass="'lh-base'">
+            <Dropdown class="'lh-1'" togglerClass="'d-print-none btn p-0 ms-1 lh-sm border-0'" menuClass="'lh-base'">
```

Comment:
Is this change meant to solve what exactly? The overlap with "Set Status"?

Reply:
No, the flickering when you select/unselect a line in the list view. 

When you select a line, the kog disappears and then it creates this flickering because the kog is 19px high and the breadcrum title alone is 17px
![Screenshot 2023-05-05 at 17 15 19](https://user-images.githubusercontent.com/110090660/236498417-a05da913-2cf2-4999-95f6-30cf45b9e31b.png)

![Screenshot 2023-05-05 at 17 15 37](https://user-images.githubusercontent.com/110090660/236498479-14891143-d42b-4ccd-b712-e1e553ff3a2a.png)